agent/util-scripts/pbench-tool-meister-start

```diff
@@ -995,8 +1007,8 @@ if __name__ == "__main__":
     parser.add_argument(
         "--orchestrate",
         dest="orchestrate",
-        default="create",
-        choices=("create", "existing"),
+        default=os.environ.get("PBENCH_ORCHESTRATE", _def_orchestrate_choice),
+        choices=sorted(_orchestrate_choices),
```

Comment:
Perhaps it would be handy to ensure that `_orchestrate_choices` is already sorted, since two of the places where it is used it has to be sorted.